misc/test_ping

```diff
@@ -5,20 +5,37 @@ RESULT_LINES=/tmp/result_lines.txt
 MONO_LOG=/tmp/mono-log.txt
 MANIFEST=./module_manifest.json
 REPORT_FILE=/tmp/report.txt
+status=pass
 
 echo Baseline ping test report | tee -a $MONO_LOG
+ls -l 
+if [ ! -f /scans/startup.pcap ]; then
+    exit 1
```

Comment:
Does this print any diagnostic?  Maybe add a "missing /scans/startup.pcap" message or something